tests/unit/test_utils.py

```diff
@@ -88,3 +90,30 @@ def test_get_vlan_tags_and_masks(self):
         for vlan_range, expected in zip(vlan_ranges, expecteds):
             with self.subTest(range=vlan_range, expected=expected):
                 assert get_vlan_tags_and_masks(*vlan_range) == expected
+
+    async def test_check_disabled_component(self):
+        """Test check disabled component"""
+        uni_a = MagicMock()
+        uni_a.interface.switch = "00:01"
+        uni_a.interface.switch.status = EntityStatus.DISABLED
```

Comment:
Looks like `async def` is not catching an exception, we typically only use an `async def test_` when there's dependency on a coroutine or event_loop internally, in this case here it doesn't look like `async def` is needed. 

- Can you remove the async def and turn it into a standard test func? and fix `AttributeError: 'str' object has no attribute 'status'`
- Also, if you could cover the case where it doesn't raise the exception here too it'd be great just so the intention is the business logic validation is entirely captured in the test case, which will be helpful for future readers especially not familiar with this part yet.

Reply:
Fixed and added the extra coverage.